src/spinegeneric/cli/check_data_consistency.py

```diff
@@ -47,6 +38,62 @@ def read_tsv(path: Path) -> tuple[list[Fieldname], list[Row]]:
         return reader.fieldnames, list(reader)
 
 
+def validate_whitespace(value: str):
+    if value != value.lstrip():
+        raise ValueError("should not have leading whitespace")
+    if value != value.rstrip():
+        raise ValueError("should not have trailing whitespace")
+
+
+def validate_integer(value: str) -> int:
+    validate_whitespace(value)
+    try:
+        return int(value)
+    except ValueError:
+        raise ValueError("should be an integer") from None
+
+
+def validate_sex(value: str):
+    validate_whitespace(value)
+    if value not in ["M", "F", "O"]:
+        raise ValueError("should be one of 'M', 'F', 'O', or 'n/a'")
```

Comment:
I notice the use of `n/a` in the error message, but missing from the check itself. Is this intentional? (i.e. will `n/a` get parsed as `None` and/or skipped?)